scripts/linters/css_linter.py

```diff
@@ -136,3 +136,20 @@ def perform_all_lint_checks(self):
             return []
 
         return self._lint_css_files()
+
+
+def get_linters(config_path, files_to_lint, verbose_mode_enabled=False):
+    """Creates ThirdPartyCSSLintChecksManager and returns it.
+
+    Args:
+        config_path: str. Path to the configuration file.
+        files_to_lint: list(str). A list of filepaths to lint.
+        verbose_mode_enabled: bool. True if verbose mode is enabled.
+
+    Returns:
+        tlinter: object(linter). Returns linter object.
```

Comment:
Wrong typeinfo. Should be "ThirdPartyCSSLintChecksManager.".

Ditto in other files. If you find it useful to declare a "LinterManager" superclass, you can, and then have all the linters inherit from it.

Reply:
Done.